src/views/transaction/components/transaction-overview/transaction-overview.view.jsx

```diff
@@ -58,6 +67,12 @@ function TransactionOverview ({
     )
   }
 
+  function getEstimatedWithdrawFee () {
+    return estimatedWithdrawFeeTask.status === 'successful'
+      ? getAmountInPreferredCurrency(estimatedWithdrawFeeTask.data, preferredCurrency, fiatExchangeRates).toFixed(2)
+      : '--'
```

Comment:
Do you want do have any error displayed to the user here or not?

Reply:
Nope!